tests/e2e/default-mobile/enketo/training-cards.wdio-spec.js

```diff
@@ -0,0 +1,133 @@
+const utils = require('@utils');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const trainingCardsPage = require('@page-objects/default/enketo/training-cards.wdio.page');
+const placeFactory = require('@factories/cht/contacts/place');
+const userFactory = require('@factories/cht/users/users');
+const personFactory = require('@factories/cht/contacts/person');
+const commonElements = require('@page-objects/default/common/common.wdio.page');
+const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
+const privacyPolicyFactory = require('@factories/cht/settings/privacy-policy');
+const privacyPage = require('@page-objects/default/privacy-policy/privacy-policy.wdio.page');
+const commonEnketoPage = require('@page-objects/default/enketo/common-enketo.wdio.page');
+
+describe('Training Cards', () => {
+
+  const expectedConfirmMessage = 'This training is not finished. ' +
+    'If you leave now, you will lose your progress and be prompted again later to complete it.';
+
+  const formDocId = 'training:training-cards-text-only';
+
```

Comment:
```suggestion
describe('Training Cards', () => {
  const expectedConfirmMessage = 'This training is not finished. ' +
    'If you leave now, you will lose your progress and be prompted again later to complete it.';
  const formDocId = 'training:training-cards-text-only';

```

Reply:
The form title is optional. Let me check the code history and get back to you in a few

Reply:
[In 2017, all form titles](https://github.com/medic/cht-core/commit/5ddb2ad07e71baa49b2354775bc633b413e2d7b1) are hidden because the title text is already in the top navigation bar of the mobile screen (no point showing the same title twice)
<img width="491" alt="Screenshot 2024-10-09 at 1 56 14 PM" src="https://github.com/user-attachments/assets/49c9520a-ee83-4106-b792-7ca6a782246b">

That's not the case for training cards, I'm opening a small fix for that and add you as reviewer

Reply:
@kapelo here it is:
https://github.com/medic/cht-core/pull/9518